Yoast/Sniffs/WP/RequiredParametersSniff.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * YoastCS\Yoast\Sniffs\Files\FileNameSniff.
+ *
+ * @package Yoast\YoastCS
+ * @author  Jip Moors
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace YoastCS\Yoast\Sniffs\WP;
+
+use WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * Verifies that optional parameters are set, when they should be set.
```

Comment:
`, when they should be set` is redundant. These parameters are _optional_ as stated in the first part of the sentence, so there is no _should be set_, other than that the YoastCS standard wants them to be, which is what the sniffs is all about in the first place.